pyproject.toml

```diff
@@ -0,0 +1,54 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
+name = "astro-image-display-api"
+dynamic = ["version"]
+description = 'Astronomical image display API definition for cross-backend interoperability'
+readme = "README.md"
+requires-python = ">=3.11"
+license = "BSD-3-Clause"
+keywords = []
+authors = [
+  { name = "Matt Craig", email = "mattwcraig@gmail.com" },
+]
+classifiers = [
+  "Development Status :: 4 - Beta",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
```

Comment:
```suggestion
  "Programming Language :: Python :: 3.12",
  "Programming Language :: Python :: 3.13",
```

Reply:
Good idea; I also dropped Python 3.11 because I got bit yesterday by the difference between Python >=3.12 parsing of f-strings and the earlier version of parsing.